scripts/monitor_lambdas.py

```diff
@@ -75,28 +105,57 @@ def send_slack_post(webhook:str, stages:dict):
         )
 
 
-def get_webhook_ssm(secret_name=None):
-    stage = os.environ['DSS_DEPLOYMENT_STAGE']
-    secrets_store = os.environ['DSS_SECRETS_STORE']
-    if secret_name is None:
-        secret_name = 'monitor-webhook'
-    secret_id = f'{secrets_store}/{stage}/{secret_name}'
-    res = secretsmanager.get_secret_value(SecretId=secret_id)
-    return res['SecretString']
-
+def format_data_size(value: int):
+    base = 1000
+    bytes = float(value)
+    suffix = ('kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB')
+    if bytes < base:
+        return '%d Bytes' % bytes
+    for i, s in enumerate(suffix):
+        unit = base ** (i + 2)
+        if bytes < unit:
+            return (format + ' %s') % ((base * bytes / unit), s)
+        elif bytes < unit:
+            return (format + '%s') % ((base * bytes / unit), s)
+
+# conditionals
 if os.environ["DSS_DEPLOYMENT_STAGE"] is None:
     raise ValueError('Missing DSS_DEPLOYMENT_STAGE, exiting....')
     exit(1)
 
-stages = {f'{os.environ["DSS_DEPLOYMENT_STAGE"]}': None }
+# variables
+script_end_time = datetime.datetime.now()
```

Comment:
Is this really the `script_end_time`?

Reply:
ah i should rename that, for the AWS request, we would have to send the time period we want to look at, (starttime -> endtime)

Reply:
renamed